.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -3,16 +3,10 @@
 <!--
 Using the template:
 
- 1. Leave all headlines in place
- 2. Replace instructional texts with your own words
+ 1. Replace instructional texts with your own words
+ 2. Remove headlines when not applicable
  3. Tick of completed checklist items as you complete them
- 4. If you intentionally skip a checklist item, see below instruction
-
-Skipping items in checklists:
-
-Tick the item checkbox, ~strikethrough item text~, and write why it was skipped, example:
-
-- [x] ~Skipped item~ This is a documentation fix
+ 4. Feel free to skip checklist items that aren't applicable
```

Comment:
Shouldn't it say *how* to skip items in the check list since that's the problem we've been having? I don't like the "feel free" part where there's no requirement of making it evident to others if something was skipped...

Reply:
In general, I've found the content & context of the PR makes it self-evident when something can be skipped

Reply:
I don't have a mental recollection of the check list when I review, so if someone just deletes an item, I will just have an odd feeling that something isn't right.

I worry mostly about the deletion of items. But I also worry that skipping them and leaving them unchecked "by design" will create a new practice of just calling shots when merging -- something like "that's probably supposed to be skipped". Once we head down that road, we're going to have problems, because the check lists loose their legitimacy.

Reply:
I'd like to find something that allows for both quick and streamlined flows, but also encourages more rigor and process when appropriate.

* Some PRs are literally a single line change with obvious implications
* Some PRs are big, consequential changes that require meticulous review and process

IMO the reviewer and submitter should be allowed some flexibility to move quickly through the process when appropriate, but also demand more info when not appropriate.

Agreed that perhaps deleting sections is not good, because it might make it harder to move from simple to more complex. I'll remove that guidance